CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 cmake_minimum_required(VERSION 3.10)
 project(roboteam_utils VERSION 0.1 DESCRIPTION "utils for Roboteam Twente")
 
-set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD 20)
```

Comment:
This is what I mean with CMake hacking.

What this does, is that it changes this variable in the CMake for ALL repositories (unless another repository changes it somewhere else, again). So this will also set Roboteam_AI's cxx standard to 20.

Reply:
I know it was 17 before, but this is probably not advisable.

Reply:
Right, so how would you recommend us to do this.

Reply:
target_compile_options(), or set this in our top-level CMakeLists in Roboteam_Suite

Reply:
But probably for now don't touch it yet. We should first do the system refactor and then change this, IMO.

Reply:
okay then merge?